### PR TITLE
Add source for local-yum-repo image used in tests

### PR DESCRIPTION
**Pull request by bboreham**
This was omitted when the repo was open-sourced.

### CODE REVIEW

File: environments/local-rpm-repo/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    name=Local
+    baseurl=http://localhost:8080
+    enabled=1
+    gpgcheck=0
+    ```
+
+5. Configure it in your `cluster.yaml`:
+
+    ```yaml
+    apiVersion: "cluster.k8s.io/v1alpha1"
```
Comment (luxas) [inline environments/local-rpm-repo/README.md:48]: We might want to upgrade this to v1alpha3 now

File: environments/local-rpm-repo/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            files:
+            - source:
+                configmap: repo
+                key: local.repo
+              destination: /etc/yum.repos.d/local.repo
+    ```
+
+## Future work
+
+The exactly list of packages to install should be driven by the WKS v2 plan.
```
Comment (luxas) [inline environments/local-rpm-repo/README.md:63]: Maybe remove this comment?

File: environments/local-rpm-repo/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    && \
+    yum clean all
+
+RUN mkdir -p /var/www/html/repos/{base,centosplus,extras,updates}
+COPY docker-ce.repo /etc/yum.repos.d/docker-ce.repo
+COPY kubernetes.repo /etc/yum.repos.d/kubernetes.repo
+RUN yum --downloadonly --downloaddir /var/www/html/repos install -y --disableexcludes=kubernetes \
+    device-mapper-persistent-data \
+    docker-ce-19.03.8 \
+    kubeadm-1.16.11 \
```
Comment (luxas) [inline environments/local-rpm-repo/Dockerfile:15]: is it possible to install the same package of different versions?

Comment (bboreham) [reply environments/local-rpm-repo/Dockerfile:15]: Yes; there's some tradeoff in image size, so adding versions we don't use can slow down CI.
Suggest we go with the ones we need for now and you can build your own image locally to see how things change.

**Pull request merged**